src/locales/en/translation.json

```diff
@@ -0,0 +1,99 @@
+{
+    "progressBar": "Progress Complete",
+    "preloadTrial": {
+      "messageText": "Please wait while the experiment loads. This may take a few minutes."
+    },
+    "fullScreenTrial": {
```

Comment:
Could you incorporate the translations? It looks like it's just the prompt for the videos and fullscreen trial. Don't worry about the getUserData trial translations for now. You can name it whatever you feel like. 

Ex.
// trasnlation.json
{
  "stimulusPrompt": "Press space bar or whatever"
}

// ibi.js

...trial,
prompt: () => i18next.t("stimulusPrompt")

Reply:
When using i18 you also need to import the file so anywhere you use it you need to import:
import i18next from 'i18next'
import '../i18n'     // relative path to the file.